pkg/pipeline/transform/transform_generic.go

```diff
@@ -33,21 +35,44 @@ func (g *Generic) Transform(input []config.GenericMap) []config.GenericMap {
 	output := make([]config.GenericMap, 0)
 	for _, entry := range input {
 		outputEntry := make(config.GenericMap)
-		for _, transformRule := range g.Rules {
+		for _, transformRule := range g.rules {
 			log.Debugf("transformRule = %v", transformRule)
 			outputEntry[transformRule.Output] = entry[transformRule.Input]
 		}
+		if g.maintain {
```

Comment:
can you just change https://github.com/netobserv/flowlogs-pipeline/pull/156/files#diff-28f2a1789a319594e5a43929c866c072ac0e053716b9ba62a7158816d41621c8R35 to be a copy of input and not from an empty GenericMap